data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/orgdetails/Entity.java

```diff
@@ -0,0 +1,45 @@
+package gov.cdc.etldatapipeline.organization.model.dto.orgdetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgElastic;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgReporting;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Entity implements OrgExtension {
+    private Long entityUid;
+    private String typeCd;
+    private String recordStatusCd;
+    private String rootExtensionTxt;
+    private String entityIdSeq;
+    private String assigningAuthorityCd;
+
+    public <T> T updateOrg(T org) {
+        if (org.getClass() == OrgReporting.class) {
+            OrgReporting orgReporting = (OrgReporting) org;
+            orgReporting.setQuickCode(rootExtensionTxt);
+            //ToDo: Revisit logic
+            orgReporting.setFacilityId(rootExtensionTxt);
+            //ToDo: Revisit logic
+            orgReporting.setFacilityIdAuth(rootExtensionTxt);
```

Comment:
I believe this should be the ASSIGNING_AUTHORITY_CD once the code filters pass through for FacilityId

Reply:
There is also an SRTE join with this information. I will update that additional logical piece in the sproc. Null if none of the conditions are met, else value for FACILITY_ID_AUTH.

Reply:
Updated.